x-pack/plugins/canvas/public/components/workpad_header/control_settings/control_settings.scss

```diff
@@ -0,0 +1,3 @@
+.canvasControlSettings__popover {
```

Comment:
This is just a replacement for `refresh_control.scss`, the style on the base class that used to be there wasn't used, nothing had that class.